app/V1Module/security/ACL/IGroupPermissions.php

```diff
@@ -31,4 +31,6 @@ function canAssignExercise(Group $group, Exercise $exercise): bool;
   function canCreateExercise(Group $group): bool;
   function canViewPublicDetail(Group $group): bool;
   function canAddStudentToArchivedGroup($group, $user): bool;
+  function canAttachExercise(Group $group): bool;
+  function canDetachExercise(Group $group): bool;
```

Comment:
Isn't this the same as canAttachGroup? I would say these permissions should be symmetrical.

Reply:
yes it is, we can consolidate this into one permission group, will do